src/components/Disputes/DisputeCurrentRuling.js

```diff
@@ -55,42 +55,81 @@ function DisputeCurrentRuling({ dispute }) {
           item: juryOutcomeToString(outcome),
           percentage: weight,
         }))}
-        renderFullLegendItem={({ color, item, index, percentage }) => (
-          <div
-            css={`
-              display: flex;
-              align-items: center;
-            `}
-          >
-            <div
-              css={`
-                background: ${color};
-                width: 8px;
-                height: 8px;
-                margin-right: 8px;
-                border-radius: 50%;
-              `}
-            />
+        renderFullLegendItem={({ color, item, index, percentage }) => {
+          // We'll show the juror voting weight hint only if the juror participated in the  current ruling
+          // and has been drafted more than once
+          const showMyVotingWeight = index === 0 && myWeight > 1
```

Comment:
Not sure if this was discussed with @dizzypaty, but I think this information is useful to everyone involved, if there is a round where someone was drafted more than once.

We could change the text to be this if you are the actual juror who got drafted more than once though.

Reply:
Yeah good point!